### PR TITLE
Update TNT test to cover eventCreatedDateTime + secret for the subscription model

### PR DESCRIPTION
**Pull request by nt-as**

### CODE REVIEW

File: src/test/resources/tnt/v2/EquipmentEventSchema.json [hunk 1]
```diff
@@ -27,6 +27,11 @@
       "title": "The eventType schema",
       "default": ""
     },
+    "eventCreatedDateTime": {
+      "type": "string",
+      "title": "The eventDateTime schema",
```
Comment (HenrikHL) [inline src/test/resources/tnt/v2/EquipmentEventSchema.json:32]:
```suggestion
      "title": "The eventCreatedDateTime schema",
```

File: src/test/resources/tnt/v2/EquipmentEventsSchema.json [hunk 2]
```diff
@@ -36,6 +36,11 @@
             "title": "The eventType schema",
             "default": ""
           },
+          "eventCreatedDateTime": {
+            "type": "string",
+            "title": "The eventDateTime schema",
```
Comment (HenrikHL) [inline src/test/resources/tnt/v2/EquipmentEventsSchema.json:41]:
```suggestion
            "title": "The eventCreatedDateTime schema",
```

File: src/test/resources/tnt/v2/EventSchema.json [hunk 3]
```diff
@@ -1,9 +1,40 @@
...
+      "default": ""
+    },
+    "eventType": {
+      "type": "string",
+      "title": "The eventType schema",
+      "default": ""
+    },
+    "eventCreatedDateTime": {
+      "type": "string",
+      "title": "The eventDateTime schema",
```
Comment (HenrikHL) [inline src/test/resources/tnt/v2/EventSchema.json:19]:
```suggestion
      "title": "The eventCreatedDateTime schema",
```

File: src/test/resources/tnt/v2/EventsSchema.json [hunk 4]
```diff
@@ -1,10 +1,41 @@
...
+      "default": ""
+    },
+    "eventType": {
+      "type": "string",
+      "title": "The eventType schema",
+      "default": ""
+    },
+    "eventCreatedDateTime": {
+      "type": "string",
+      "title": "The eventDateTime schema",
```
Comment (HenrikHL) [inline src/test/resources/tnt/v2/EventsSchema.json:20]:
```suggestion
      "title": "The eventCreatedDateTime schema",
```

File: src/test/resources/tnt/v2/ShipmentEventSchema.json [hunk 5]
```diff
@@ -26,6 +26,11 @@
       "title": "The eventType schema",
       "default": ""
     },
+    "eventCreatedDateTime": {
+      "type": "string",
+      "title": "The eventDateTime schema",
```
Comment (HenrikHL) [inline src/test/resources/tnt/v2/ShipmentEventSchema.json:31]:
```suggestion
      "title": "The eventCreatedDateTime schema",
```

File: src/test/resources/tnt/v2/EventsSchema.json [hunk 6]
```diff
@@ -1,10 +1,41 @@
...
+      "default": ""
+    },
+    "eventDateTime": {
+      "type": "string",
+      "title": "The eventDateTime schema",
```
Comment (nt-as) [inline src/test/resources/tnt/v2/EventsSchema.json:25]:
```suggestion
       "title": "The eventCreatedDateTime schema",
```

File: src/test/resources/tnt/v2/ShipmentEventSchema.json (hunk 5 above)
Comment (nt-as) [inline src/test/resources/tnt/v2/ShipmentEventSchema.json:31]:
```suggestion
      "title": "The eventCreatedDateTime schema",
```

File: src/test/resources/tnt/v2/ShipmentEventsSchema.json [hunk 7]
```diff
@@ -35,6 +35,11 @@
             "title": "The eventType schema",
             "default": ""
           },
+          "eventCreatedDateTime": {
+            "type": "string",
+            "title": "The eventDateTime schema",
```
Comment (nt-as) [inline src/test/resources/tnt/v2/ShipmentEventsSchema.json:40]:
```suggestion
      "title": "The eventCreatedDateTime schema",
```

File: src/test/resources/tnt/v2/TransportEventSchema.json [hunk 8]
```diff
@@ -27,6 +27,11 @@
       "title": "The eventType schema",
       "default": ""
     },
+    "eventCreatedDateTime": {
+      "type": "string",
+      "title": "The eventDateTime schema",
```
Comment (HenrikHL) [inline src/test/resources/tnt/v2/TransportEventSchema.json:32]:
```suggestion
      "title": "The eventCreatedDateTime schema",
```

File: src/test/resources/tnt/v2/TransportEventsSchema.json [hunk 9]
```diff
@@ -36,6 +36,11 @@
             "title": "The eventType schema",
             "default": ""
           },
+          "eventCreatedDateTime": {
+            "type": "string",
+            "title": "The eventDateTime schema",
```
Comment (HenrikHL) [inline src/test/resources/tnt/v2/TransportEventsSchema.json:41]:
```suggestion
            "title": "The eventCreatedDateTime schema",
```

File: src/test/resources/tnt/v2/TransportEventSchema.json (hunk 8 above)
Comment (nt-as) [inline src/test/resources/tnt/v2/TransportEventSchema.json:32]:
```suggestion
      "title": "The eventCreatedDateTime schema",
```

File: src/test/resources/tnt/v2/TransportEventsSchema.json (hunk 9 above)
Comment (nt-as) [inline src/test/resources/tnt/v2/TransportEventsSchema.json:41]:
```suggestion
      "title": "The eventCreatedDateTime schema",
```

**Review by HenrikHL — DISMISSED** [review]

File: src/test/resources/tnt/v2/EventsSchema.json (hunk 4 above)
Comment (HenrikHL) [inline src/test/resources/tnt/v2/EventsSchema.json:20]: still not solved in my end?!?

File: src/test/resources/tnt/v2/EventsSchema.json [hunk 10]
```diff
@@ -1,10 +1,41 @@
...
+      "default": ""
+    },
+    "eventCreatedDateTime": {
+      "type": "string",
+      "title": "The eventCreatedDateTime schema",
+      "default": ""
+    },
+    "eventDateTime": {
+      "type": "string",
+      "title": "The eventCreatedDateTime schema",
```
Comment (HenrikHL) [inline src/test/resources/tnt/v2/EventsSchema.json:25]: Needs to be eventDateTime

Comment (nt-as) [inline src/test/resources/tnt/v2/EventsSchema.json:25]:
```suggestion
      "title": "The eventDateTime schema",
```

**Review by HenrikHL — APPROVED** [review]

**Pull request merged**